cmd/hiCarOverlap/hiCarOverlap.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+	"strings"
+)
+
+func hiCarOverlap(geneList string, hiCarData string, outFile string) {
+	var hiCarPairs []string = fileio.Read(hiCarData)
+	var genes []string = fileio.Read(geneList)
+	var err error
+	out := fileio.EasyCreate(outFile)
+
+	hiCarGeneMap := make(map[string]int)
+
+	for i, line := range genes {
+		hiCarGeneMap[line] = i
+	}
+	for _, hiCarCont := range hiCarPairs {
+		items := strings.Split(hiCarCont, "\t")
+		_, ok := hiCarGeneMap[items[9]]
+		_, ok2 := hiCarGeneMap[items[10]]
+		if ok || ok2 {
+			_, err = fmt.Fprintf(out, hiCarCont+"\n")
+			exception.PanicOnErr(err)
+		}
```

Comment:
i recommend an else fatalf here if !ok or !ok2 and it should report what each of them is so you dont have to track back which element is missing without extra context.